modules/smart-agent_pm2/conf/readme.yaml

```diff
@@ -0,0 +1,3 @@
+documentations:
```

Comment:
could be good to add some information how to setup pm2 metrics with an example of agent configuration to fetch them.

also it seems useful to explain why `(not filter('name', 'pm2-metrics'))` is required